config_example.js

```diff
@@ -25,5 +25,17 @@ module.exports = {
     },
     newRelic: {
         licenseKey: process.env.newRelicLicenseKey || ''
+    },
+    installed: {
+        client_id: "397600756791-ekv992pp3cugmplk8ttk7g1al3up88lk.apps.googleusercontent.com",
```

Comment:
Remove all the keys from `config_example.js`. Only write keys here and values as blank. The values can be added in the `config.js` file.